File: Python3/0168. Excel Sheet Column Title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        res = ""
        while columnNumber != 0:
            temp = (columnNumber - 1) % 26 + 1
            res = chr(temp - 1 + ord("A")) + res
            columnNumber = (columnNumber - temp) // 26
        
        return res

File: C++/0168. Excel Sheet Column Title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res = "";
        while(columnNumber != 0) {
            int temp = (columnNumber - 1) % 26 + 1;
            res = char(temp - 1 + 'A') +res;
            columnNumber = (columnNumber - temp) / 26;
        }
        return res;
    }
};
